### PR TITLE
[DOCS] Fixing lexer issues

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Fixing lexers for code-blocks.

### CODE REVIEW

File: demos/python_demos/llm_text_generation/README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ python3 client_unary.py --url localhost:9000 \
 ```
 
 Example output:
-```bash
```
Comment (dkalinowski) [inline demos/python_demos/llm_text_generation/README.md:160]: Removing bash keyword will remove automatic testing feature @ngrozae

File: docs/custom_node_development.md [hunk 2]
```diff
@@ -175,9 +175,9 @@ Below you can see the list of fully functional custom nodes embedded in the mode
 | [face blur custom node](https://github.com/openvinotoolkit/model_server/tree/main/src/custom_nodes/face_blur) | `/ovms/lib/custom_nodes/libcustom_node_face_blur.so`|
 
 
-**Example:** 
+**Example:**
 Including built-in [horizontal OCR custom node](https://github.com/openvinotoolkit/model_server/tree/main/src/custom_nodes/horizontal_ocr) in the `config.json` would look like:
-```json
```
Comment (mzegla) [inline docs/custom_node_development.md:180]: Why do we remove this?

File: docs/python_support/reference.md [hunk 3]
```diff
@@ -17,7 +17,7 @@
 The publicly available `openvino/model_server` image on Docker Hub supports Python, but does not come with external modules installed. If Python is all you need then you can use the public image without modification. Otherwise, you will need to extend the public image with additional layers that install any modules required for your Python code to run. For example, let's say your code requires numpy.
 In that case, your Dockerfile may look like this:
 
-```dockerfile
```
Comment (mzegla) [inline docs/python_support/reference.md:20]: Why it needs to be removed?

File: docs/custom_node_development.md (hunk 2 above)
Comment (sgolebiewski-intel) [reply docs/custom_node_development.md:180]: To avoid this issue:

image

File: docs/python_support/reference.md (hunk 3 above)
Comment (mzegla) [reply docs/python_support/reference.md:20]: Is it something that will be fixed in the future? It looks a bit better in markdown with that, so I'd prefer it to stay in the long term.

**Comment by sgolebiewski-intel** [comment]
Closing in favor of this PR: https://github.com/openvinotoolkit/openvino/pull/23952 which fixes the issues in highlighting the code-blocks

**Pull request closed without merging**